### PR TITLE
Moving user-facing messages to BotResponses

### LINKED ISSUE #9: User-facing messages should be stored in a static class
Frameworks like the Android app development system tend to store all user-facing text in a key-value store. This makes it easier to implement translations and to quickly update messages. @peterkos suggested a similar system: generate the `$help` embed and error messages from a static class to keep most/all of the user-facing text in one place.

# Tasks
- [ ] Create a `BotResponses` static class that contains a static getter for each message. Look at [authbot's `BotRepsonses.java`](https://github.com/codeRIT/authbot-hm/blob/develop/src/main/java/info/BotResponses.java) for reference.
- [ ] Replace all existing message strings and embed builders calls to BotResponses static getters

### PR DESCRIPTION
**Pull request by sjmiller7**
And also fixing the toLowerCase issue

fixes #9

### CODE REVIEW

File: src/main/java/info/BotResponses.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    /**
+     * Reporting that the queue is empty
+     *
+     * @param channel The text channel to send message to
+     * @param member The member to ping/mention
+     * @param topic The topic queue that is empty
+     */
+    public static void queueIsEmpty(TextChannel channel, Member member, Topic topic) {
+        channel.sendMessage(String.format(
+                    "%s Queue \"%s\" is empty.",
```
Comment (skyegallup) [inline src/main/java/info/BotResponses.java]: indentation (apply to ALL of them)

File: src/main/java/listeners/MainEventListener.java [hunk 2]
```diff
@@ -95,10 +96,7 @@ private static boolean isMentor(Member member) {
     private static Optional<Topic> checkTopicExists(Member member, TextChannel channel, Server server, String topicName) {
         Optional<Topic> optionalTopic = server.getTopic(topicName.toLowerCase());
         if (optionalTopic.isEmpty()) {
-            channel.sendMessage(String.format(
-                    "%s Topic \"%s\" does not exist.",
-                    member.getAsMention(),
-                    topicName)).queue();
+            BotResponses.noSuchTopic(channel, member, topicName);
```
Comment (skyegallup) [inline src/main/java/listeners/MainEventListener.java:99]: this is bad, but Colby will fix later, don't worry

**Review by skyegallup — CHANGES_REQUESTED** [review]
fix indentation please

File: src/main/java/info/BotResponses.java [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+import entities.Topic;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.TextChannel;
+
+/**
+ * Responses that the bot gives the user
+ */
+public class BotResponses {
+    /**
+     * Telling the user they do not have permission for a command
```
Comment (peterkos) [inline src/main/java/info/BotResponses.java]: nit: tense ("Tells the user" vs. "Telling the user")

File: src/main/java/info/BotResponses.java (hunk 1 above)
Comment (peterkos) [reply src/main/java/info/BotResponses.java]: ^^^

File: src/main/java/listeners/MainEventListener.java (hunk 2 above)
Comment (peterkos) [reply src/main/java/listeners/MainEventListener.java:99]: Colby fix! Colby fix fast! Good code fix by Colby!

File: src/main/java/listeners/MainEventListener.java [hunk 4]
```diff
@@ -241,18 +224,13 @@ private void ready(Member member, TextChannel channel, Server server, String[] a
         // do not run if caller does not have mentor role for this topic or admin privileges
         Topic topic = optionalTopic.get();
         if (!isMentor(member, topic) && !isAdmin(member)) {
-            channel.sendMessage(member.getAsMention() + " You do not have permission to run this command.").queue();
+            BotResponses.noPermission(channel, member);
```
Comment (peterkos) [inline src/main/java/listeners/MainEventListener.java:227]: gotta love the git diff here with `)` lol

**Review by peterkos — DISMISSED** [review]

**Review by skyegallup — APPROVED** [review]

**Pull request merged**